services/conference/src/scripts/components/map/map.tsx

```diff
@@ -2,16 +2,20 @@ import {Base} from '@components/map/Base'
 import {ParticipantsLayer} from '@components/map/ParticipantsLayer'
 import {ShareLayer} from '@components/map/ShareLayer'
 import {BaseProps} from '@components/utils'
+import {useStore as useContentsStore} from '@hooks/SharedContentsStore'
+import {useObserver} from 'mobx-react-lite'
 import React from 'react'
 import {BackgroundLayer} from './BackgroundLayer'
-import {StoreProvider as ParticipantsProvider} from '@hooks/ParticipantsStore'
-import participantsStore from '@stores/participants/Participants'
+
 
 export const Map: React.FC<BaseProps> = (props) => {
+  const store = useContentsStore()
+  const stream = useObserver(() => store.mainStream)
+
   return (
     <Base {...props}>
-      <BackgroundLayer />
-      <ShareLayer />
+      {<BackgroundLayer isTransparnet={stream !== undefined} />}
+      {stream ? <div /> : <ShareLayer />}
```

Comment:
`{stream ? undefined : <ShareLayer />}` seems better.

Reply:
I found undefined causes compile error.

Reply:
So I have to keep <div />. Is there a better solution ?

Reply:
Compile error is caused by child type in BaseProps. I've fixed, please refer to: e387f35d0fde75190722b2308bb417cb0745f743